charger/easee.go

```diff
@@ -674,7 +674,7 @@ var _ api.Meter = (*Easee)(nil)
 
 // CurrentPower implements the api.Meter interface
 func (c *Easee) CurrentPower() (float64, error) {
-	if status, err := c.Status(); err != nil || status != api.StatusC {
+	if status, err := c.Status(); err != nil || status == api.StatusA {
```

Comment:
Warum? Auch in B lädt das Auto ja nicht. Wir verschlucken zwar Standbyleistung, jetzt würde aber in B dennoch auch ein grob falscher Wert ausgegeben?

Reply:
Da gab es von Robbe64 den [Hinweis](https://github.com/evcc-io/evcc/issues/10175#issuecomment-1750203827), daß er auch Status B mit laufender Ladung gesehen hat, und allcoolusernamesaregone hat das auch bestätigt.

Ich bin unsicher ob das jetzt etwas verschlimmert, oder egal ist.
Wo ich mir sicher bin ist, daß der Bug den UlrichSchreiner gemeldet hat (Charge Power wird bei nicht verbundenem Fahrzeug > 0 angezeigt) hiermit gefixt wird.
Das ist hier also eigentlich erstmal nur die Minimaländerung die nichts kaputt macht.

Reply:
Das heisst ja im Klartext, dass der Fix nur ein halber ist. Dann können wir es eigentlich auch gleich lassen?

Reply:
Nein. Es heisst das dieser Change den gemeldeten Fehler adressiert. Dieser ist auch nach meinem Verständnis nicht rein optisch/kosmetisch, weil ein Loadpoint mit falscher ChargePower Einfluß auf andere Komponenten hat (weitere Loadpoints, aber auch nachgelagerte Steuerungen die über MQTT & Co angebunden sind). Genau dies hatte der OP auch so gemeldet.

> Allerdings führt dieser Bug aber auch dazu, dass die angezeigten Daten von evcc nun [...] nutzlos sind. Es wäre schick, wenn [...] die Berechnung für andere Ladepunkte (ausser der easee) noch korrekt funktioniert.

Wenn Du diesen Fehler als nicht relevant ansiehst, sei bitte so nett und hake Dich in die Diskussion mit ein, damit wir den Sollzustand für diesen PR mit den Usern klären können.